source/configuration_manual/lazy_expunge_plugin.rst

```diff
@@ -72,132 +110,113 @@ Example configuration:
 
   mail_plugins = $mail_plugins lazy_expunge
   plugin {
+    # Move expunged messages into the .EXPUNGED namespace
     lazy_expunge = .EXPUNGED/
   }
 
-namespaces (obsolete, v2.0 only)
-================================
-
-The namespaces are:
-
-1. Expunged messages namespace. Whenever a message is expunged in mailbox
-   <name>, it's moved to a mailbox <name> in this namespace. The mailboxes are
-   created automatically as needed.
-2. Deleted mailboxes namespace. Whenever a mailbox <name> is deleted, it's
-   moved here with name <name>-YYYMMDD-hhmmss. The timestamp is there so that
-   the mailbox can be deleted multiple times. If the mailbox is deleted
-   multiple times within a second, random 16bit hex value is appended to it.
-3. Expunged messages in a deleted mailbox namespace. When a mailbox is deleted
-   and it has messages in its expunged namespace, the mailbox is moved from the
-   expunged namespace to this namespace. The destination mailbox name is the
-   same as in the 2nd namespace (ie. contains the same timestamp).
-
-Example configuration:
-
-.. code-block:: none
-
-  # the default namespace
-  namespace {
-    prefix =
-    separator = /
-    inbox = yes
-  }
-
-  # namespaces for lazy_expunge plugin:
-  namespace {
-    prefix = .EXPUNGED/
-    separator = /
-    location = maildir:~/Maildir/expunged
-  }
-  namespace {
-    prefix = .DELETED/
-    separator = /
-    location = maildir:~/Maildir/deleted
-  }
-  namespace {
-    prefix = .DELETED/.EXPUNGED/
-    separator = /
-    location = maildir:~/Maildir/deleted/expunged
-  }
-
-  mail_plugins = $mail_plugins lazy_expunge
-  plugin {
-    lazy_expunge = .EXPUNGED/ .DELETED/ .DELETED/.EXPUNGED/
-  }
-
-Multi-dbox
-==========
+Multi-dbox (mdbox)
+""""""""""""""""""
```

Comment:
maybe just name it "mdbox"? I don't think it's common to know what "multi-dbox" is..

Reply:
Done, and added link to dbox documentation (still in wiki? we really need to move that).